src/reduced_order/halton.cpp

```diff
@@ -0,0 +1,896 @@
+#include <cmath>
+#include <cstdlib>
+#include <iomanip>
+#include <iostream>
+#include <ctime>
+#include "halton.h"
+
+namespace PHiLiP {
+namespace ProperOrthogonalDecomposition {
+
+using namespace std;
+
+//****************************************************************************80
+
+double *halton(int i, int m)
+
+//****************************************************************************80
+//
+//  Purpose:
+//
+//    HALTON computes an element of a Halton sequence.
+//
+//  Licensing:
+//
+//    This code is distributed under the GNU LGPL license.
```

Comment:
I'm gonna assume that you somehow can't import that code through some submodules?

Reply:
That's right, I found the code as a cpp file that I downloaded.